python-threatexchange/threatexchange/hashing/pdq_utils.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+BITS_IN_PDQ = 256
+
+
+def simple_distance_binary(bin_a, bin_b):
+    assert len(bin_a) == BITS_IN_PDQ
+    assert len(bin_b) == BITS_IN_PDQ
+    distance = 0
+    for i in range(BITS_IN_PDQ):
+        if bin_a[i] != bin_b[i]:
+            distance += 1
+    return distance
```

Comment:
pythonic code golf, feel free to ignore
`return sum(bin_a[i] != bin_b[i] for i in range(BITS_IN_PDQ))`